packages/cli/src/lib/run/log-displayer.ts

```diff
@@ -43,6 +44,12 @@ function readLogs(logplexURL: string, isTail: boolean, recreateSessionTimeout?:
       // should only land here if --tail and no error status or message
     })
 
+    // only display for fir apps
+    if (es.readyState === 0 && recreateSessionTimeout) {
+      ux.log(color.cyan.bold('Waiting for logs...'))
+      ux.log()
+    }
+
```

Comment:
in addition to the points @sbosio made, which i agree with, can we maybe open the question for what color this `waiting for logs` message should be? `cyan` i feel like might seem close to some of the actual app/router log output. maybe throw the open question in slack and ping anush and/or matt?

we may also want to ping @SandyPantsLai on the actual `waiting for logs` message. maybe we need some kind of indicator in front of it, again to distinguish from normal log output.. like → or something.

Reply:
I like that idea of adding `->` if we don't know of other precedence for this type of messaging. Coloring it would also be great, but would defer to Matt.

Would `Retrieving logs...` be ok to say here? Otherwise I'm ok with `Waiting for logs...`